src/foam/comics/v2/DAOBrowserView.js

```diff
@@ -229,9 +229,14 @@ foam.CLASS({
         return records && records.array && records.array.length != 0;
       },
       code: function() {
+        var adao;
+        if ( summaryView.selectedObjects && ! foam.Object.equals(summaryView.selectedObjects,{}) ) {
```

Comment:
@jlhughes summaryView.selectedObjects of type foam.core.Map. Is there a better way to do this check for an empty map?

Reply:
Ya the DAOBrowserView already creates the view from the ViewSpec so we need to use summaryView instead of this.summaryView. We do it this way cause we need to pass values between the filterViews and the table so unless we refactor all of this to use context i think this is the way we have to do it.

Reply:
Testing for an empty object/map is messy, so if this works, it's fine.

Reply:
ok